bin/resin-preload

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env node
+const info = require('../package.json')
+
+const USAGE = `
+  Usage: ${info.name} [options]
+
+  Options:
+
+    --app          Application ID (required)
+    --img          Disk image to preload into (required)
+    --api-token    API token (required, or api-key)
+    --api-key      API key (required, or api-token)
+    --api-host     API host (default: "https://api.resin.io")
+    --registry     Image registry host (default: "registry.resin.io")
```

Comment:
These defaults prob. should only change once #49 (move to registry2) is merged, but I've mended them anyway